examples/stm32f411ceu_mini_f411/blink/project.xml

```diff
@@ -4,6 +4,6 @@
     <option name="modm:build:build.path">../../../build/stm32f411ceu_mini_f411/blink</option>
   </options>
   <modules>
-    <module>modm:build:scons</module>
+    <module>modm:build:cmake</module>
```

Comment:
Don't remove SCons, just add CMake. The CI will then compile twice.

Reply:
It's a fact that compiler selection and optimization is controlled by any modern IDE you use and not some code generator. It's the industry standard and cannot be bypassed. Then you compile from an IDE it will execute eg. "cmake CMAKE_BUILD_TYPE=MinSizeRel ..." and it will inject the correct flags, it will be mixed with the flags from lbuild. The only way i see currently is to do some magic with "tools/build_script_generator/common.py" if both build systems must be preserved. The question is which path you want to choose.

Reply:
Well ok, if it's necessary, then you can filter out the flags here: https://github.com/modm-io/modm/blob/develop/tools/build_script_generator/cmake/module.lb#L89
That will then only impact CMake, but not SCons.

Reply:
Also: What IDE are you using and is there an overview of flags they insert?

Reply:
It's cmake there are responsible. It acts as a common interface to many compilers on the market so the IDE's don't have to know anything about specific compilers. A strong feature of cmake is you only need to tell it about the language features needed and you can forget all about compilers, versions and flags. Anyway in our case the most obvious flags is -std -Ox and any debug/release flag combinations.